lib/builders/latexmk.js

```diff
@@ -59,6 +60,13 @@ export default class LatexmkBuilder extends Builder {
       args.push(`-outdir="${outdir}"`)
     }
 
+    if (outputFormat) {
+      if (outputFormat !== 'pdf') {
+        args.splice(args.indexOf('-pdf'), 1)
+        args.push(`-${outputFormat}`)
+      }
+    }
+
```

Comment:
The more I think about it, if we either move the `const args` initialization to be after we've initialized `const outputFormat` or move the initialization of the latter to before the former, then we should be able to something like this

~~~ js
const args = [
       '-interaction=nonstopmode',
       '-f',
       '-cd',
       `-${outputFormat}`,
       '-synctex=1',
       '-file-line-error'
     ]
~~~

If it's the case that we want to do something additional tweaking when using PS or DVI output, we could also remove '-pdf' from `const args` initialization, and simply append it to the array like we do with the other special case arguments.